### PR TITLE
Rtt status

### PR DESCRIPTION
**Pull request by juslesan**
- Include RTTs of connections in status messages

- Store connection RTTs of the overlay in tracker

### CODE REVIEW

File: src/logic/Node.js [hunk 1]
```diff
@@ -344,7 +344,8 @@ class Node extends EventEmitter {
     _getStatus(tracker) {
         return {
             streams: this.streams.getStreamsWithConnections(tracker, this.trackersRing),
-            started: this.started
+            started: this.started,
+            rtts: this.protocols.nodeToNode.endpoint.getRtts()
```
Comment (harbu) [inline src/logic/Node.js]: Think you can shorten this to `this.protocols.nodeToNode.getRtts()` since you added a method there.

File: test/integration/tracker-node-status.test.js [hunk 2]
```diff
@@ -66,4 +67,30 @@ describe('check status message flow between tracker and two nodes', () => {
...
+                    expect(statusMessage.getStatus().rtts[nodeOne.opts.id]).toBeGreaterThanOrEqual(0)
+                }
+                receivedTotal += 1
+                if (receivedTotal === 2) {
+                    done()
+                }
+            })
+            nodeOne.subscribe(streamId2, 0)
+            nodeTwo.subscribe(streamId2, 0)
+        }, 2000)
```
Comment (harbu) [inline test/integration/tracker-node-status.test.js]: Btw this test passed for  me without the `setTimeout()`. So in furtherance of faster test suite execution can we leave the `setTimeout` out or is there some need for it?

Comment (juslesan) [reply test/integration/tracker-node-status.test.js]: It seems that the timeout is unnecessary. I had it there to guarantee that the first WS ping pong has passed so that the rtt value is defined. It seems that it is not necessary when there are only 2 nodes.

**Review by harbu — APPROVED** [review]

**Review by mirpo — APPROVED** [review]
